core/domain/classifier_services.py

```diff
@@ -431,6 +339,33 @@ def mark_training_job_complete(job_id):
                                            feconf.TRAINING_JOB_STATUS_COMPLETE)
 
 
+def store_classifier_data(job_id, classifier_data):
+    """Checks for the existence of the model and then updates it.
+
+    Args:
+        job_id: str. ID of the ClassifierTrainingJob domain object.
+        classifier_data: str. The classification model which needs to be stored
+            in the job.
+
+    Raises:
+        Exception. The ClassifierTrainingJobModel corresponding to the job_id
+            of the ClassifierTrainingJob does not exist.
+    """
+    classifier_training_job_model = (
+        classifier_models.ClassifierTrainingJobModel.get(job_id, strict=False))
+    if not classifier_training_job_model:
+        raise Exception(
+            'The ClassifierTrainingJobModel corresponding to the job_id of the'
+            'ClassifierTrainingJob does not exist.')
+
+    classifier_training_job = get_classifier_training_job_by_id(job_id)
```

Comment:
@seanlip I meant to do get_classifier_from_model(). We get a domain object for validation purposes (after updating).